pkg/pipelines/argocd/argocd.go

```diff
@@ -60,17 +60,21 @@ type resource struct {
 }
 
 const (
+	// ArgoCDNamespace is the default namespace for ArgoCD installations.
+	ArgoCDNamespace = "argocd"
+
```

Comment:
indentations don't line up.   run golint?

Reply:
This is actually the behaviour of gofmt, each group shares the same indentation.

I split the exported `ArgoCDNamespace` from the internal names, and so each group is indented separately.